tests/pact/parsing.repl

```diff
@@ -34,6 +34,11 @@
     "test module ref parsing"
     1)
 
+  (defun test-cond (a)
```

Comment:
Unless you're wanting to test `cond`-in-module you don't need a test fun

Reply:
Mostly just useful for repeated application.